vector/src/main/java/im/vector/app/features/home/HomeDetailViewModel.kt

```diff
@@ -294,7 +294,7 @@ class HomeDetailViewModel @AssistedInject constructor(@Assisted initialState: Ho
                 }
             } else {
                 Timber.d("unable to revoke invite (no pending invite)")
-                room.leave()
+                tryOrNull { room.leave() }
```

Comment:
You can remove the try catch in the `if` block (doing nothing else than throwing the failure again).
I think you should handle the failure in the `else` block or propagate it as well instead of doing nothing if the room.leave fails.